app/services/stats/r402_offender_sar_complaint_closed_cases_report.rb

```diff
@@ -0,0 +1,27 @@
+module Stats
+  class R402OffenderSarComplaintClosedCasesReport < BaseClosedCasesReport
+    class << self
+      def title
+        'Closed complaint cases report'
+      end
+
+      def description
+        'Entire list of closed complaint cases'
+      end
+
+      def etl_handler
+        Stats::ETL::OffenderSarComplaintClosedCases
+      end
+    end
+
+    def case_scope
+      CaseFinderService.new(@user).closed_cases_scope.where(type: 'Case::SAR::OffenderComplaint')
+    end
+
+    def report_type
+      ReportType.r402
+    end
+
+  end
+end
+
```

Comment:
@Polsonby In order to get this report displayable under custom report/<case type>, we need to update some flags and some fields in report_type and correspondence_types table but depending on what your thought about where you want to put this report, maybe let us have a chat?